src/libs/Localize/LocaleListener/index.desktop.js

```diff
@@ -0,0 +1,13 @@
+import ELECTRON_EVENTS from '../../../../desktop/ELECTRON_EVENTS';
+import BaseLocaleListener from './BaseLocaleListener';
+
+const listenForLocaleChanges = () => {
+// Update the system context menus with the localized options.
+    BaseLocaleListener.connect(val => window.electron.send(ELECTRON_EVENTS.LOCALE_UPDATED, val));
+};
+
+const LocaleListener = {
```

Comment:
NAB but you don't really need this – you can just do:

```js
export default {
    listenForLocaleChanges,
};
```

Or if you rename the function:

```js
export default {
    connect,
};
```